cmd/brew-test-bot.rb

```diff
@@ -721,15 +723,17 @@ def setup_formulae_deps_instances(formula, formula_name)
       build_dependencies = dependencies - runtime_or_test_dependencies
       @unchanged_build_dependencies = build_dependencies - @formulae
 
-      dependents =
-        Utils.popen_read("brew", "uses", "--include-test", "--recursive", formula_name)
-             .split("\n")
-      dependents -= @formulae
-      dependents = dependents.map { |d| Formulary.factory(d) }
+      unless ARGV.include?("--skip-dependants")
```

Comment:
Helps if I’m consistent with spelling...

```suggestion
      unless ARGV.include?("--skip-dependents")
```